packages/myst-frontmatter/src/frontmatter/validators.ts

```diff
@@ -705,29 +703,46 @@ export function validateBiblio(input: any, opts: ValidationOptions) {
  */
 export function validateThebe(input: any, opts: ValidationOptions): Thebe | undefined {
   if (input === false) return undefined;
-  if (input === true || input === 'server') return { server: true };
   if (input === 'lite') return { lite: true };
-  if (input === 'binder') return { binder: true };
+  if (typeof input === 'string' && input !== 'binder') {
+    return validationError(
+      `thebe must be a boolean, an object, "lite" or "binder", not a string: ${input}`,
+      opts,
+    );
+  }
+
+  let inputObject: Record<string, any> = input;
+  if (input === true || input === 'binder') {
+    // expand boolean methods to object
+    inputObject = { binder: true };
+  }
+
+  const value: Thebe | undefined = validateObjectKeys(inputObject, { optional: THEBE_KEYS }, opts);
 
-  const value: Thebe | undefined = validateObjectKeys(input, { optional: THEBE_KEYS }, opts);
   if (value === undefined) return undefined;
   const output: Thebe = {};
   if (defined(value.lite)) {
     output.lite = validateBoolean(value.lite, incrementOptions('lite', opts));
   }
   if (defined(value.binder)) {
-    output.binder = validateBooleanOrObject(
-      value.binder,
+    const isBoolean = validateBoolean(value.binder, {
```

Comment:
```suggestion
    const asBoolean = validateBoolean(value.binder, {
```

Maybe? Since this resolves to the boolean value, if it's `false`, then `isBoolean === false` which feels wrong?